tests/e2e/sysbench_test.go

```diff
@@ -62,13 +62,13 @@ var _ = Describe("end to end test", func() {
 				}, timeout).Should(BeTrue())
 			})
 			It("Should leave a successful job", func() {
-				pod := &batchv1.Job{}
```

Comment:
This was already done in https://github.com/xridge/kubestone/pull/111, it will conflict.